### PR TITLE
CI check for dynamic project code

### PR DESCRIPTION
**Pull request by Waldz**
Introducing CI check for up-to-date go generated code. So that we would not forget to review Tequilapi docs after endpoint changes

### CODE REVIEW

**Review by tadaskay — APPROVED** [review]

File: ci/check/check.go [hunk 1]
```diff
@@ -54,11 +56,35 @@ func CheckGoImports() error {
...
+	fmt.Println("Uncommitted files:")
+	fmt.Println(filesBefore)
+
+	mg.Deps(packages.Generate)
+
+	filesAfter, err := sh.Output("git", "status", "--short")
+	if err != nil {
+		return fmt.Errorf("could retrieve changed files: %w", err)
+	}
+	fmt.Printf("Generated files")
```
Comment (soffokl) [inline ci/check/check.go:80]:
```suggestion
	fmt.Println("Generated files:")
```

File: ci/check/check.go [hunk 2]
```diff
@@ -54,11 +56,35 @@ func CheckGoImports() error {
...
+		return fmt.Errorf("could retrieve changed files: %w", err)
 	}
+	fmt.Println("Uncommitted files:")
+	fmt.Println(filesBefore)
+
+	mg.Deps(packages.Generate)
+
+	filesAfter, err := sh.Output("git", "status", "--short")
+	if err != nil {
+		return fmt.Errorf("could retrieve changed files: %w", err)
```
Comment (soffokl) [inline ci/check/check.go:78]:
```suggestion
		return fmt.Errorf("could not retrieve changed files: %w", err)
```

File: ci/check/check.go [hunk 3]
```diff
@@ -54,11 +56,35 @@ func CheckGoImports() error {
...
+		return fmt.Errorf("could not validate swagger spec: %w", err)
+	}
+	return nil
+}
+
+// CheckGenerate checks whether dynamic project parts are updated properly.
+func CheckGenerate() error {
+	filesBefore, err := sh.Output("git", "status", "--short")
+	if err != nil {
+		return fmt.Errorf("could retrieve changed files: %w", err)
```
Comment (soffokl) [inline ci/check/check.go:69]:
```suggestion
		return fmt.Errorf("could not retrieve changed files: %w", err)
```

File: ci/check/check.go [hunk 4]
```diff
@@ -54,11 +56,35 @@ func CheckGoImports() error {
...
+
+	filesAfter, err := sh.Output("git", "status", "--short")
+	if err != nil {
+		return fmt.Errorf("could retrieve changed files: %w", err)
+	}
+	fmt.Printf("Generated files")
+	fmt.Println(filesAfter)
+
+	if filesBefore != filesAfter {
+		fmt.Println(`These files needs review with "mage generate":`)
```
Comment (soffokl) [inline ci/check/check.go:84]: You are not printing a list of files, but promising it in the message.

**Review by soffokl — APPROVED** [review]

**Pull request merged**